etc/org.opencastproject.fsresources.StaticResourceServlet.cfg

```diff
@@ -0,0 +1,4 @@
+# Configuration if the static file server shall just deliver files or if authentication shall be checked for each
```

Comment:
Maybe `Shall the static file server check authentication before delivering a file?`